apps/projects/app/components/Panel/NewProject/NewProject.js

```diff
@@ -63,160 +204,127 @@ const NewProject = () => {
     setRepoSelected(i)
   }
 
-  // if there are visible (with or tiwhout filtration) repos, show them
-  // else if there are no repos to show but filtering is active - show "no match"
-  // else there are no repos to add (possibly all that could have been added
-  // already are
-  const RepoList = ({ visibleRepos, repoArray }) => {
-    if (visibleRepos.length) return (
-      <StyledRadioList
-        items={repoArray}
-        selected={repoSelected}
-        onChange={onRepoSelected(repoArray)}
+  return (
+    <>
+      {github.status !== 'authenticated' ? (
+        <div style={{ width: '100%', textAlign: 'center', marginBottom: (3 * GU) + 'px' }}>
+          <Button
+            mode="normal"
+            onClick={handleGithubSignIn}
+          >
+            <IconGitHub
+              color={`${theme.surfaceIcon}`}
+              width="18px"
+              height="18px"
+            />
+            <span css="margin-top: 3px; margin-left: 8px">Select from GitHub</span>
+          </Button>
+        </div>
+      ) : (
+        <RepoQuery
+          onRepoSelected={onRepoSelected}
+          repoSelected={repoSelected}
+          setRepoSelected={setRepoSelected}
+        />
+      )}
+      <Button
+        mode="strong"
+        wide
+        onClick={handleNewProject}
+        disabled={repoSelected < 0}
+      >
+        Submit
+      </Button>
+    </>
+  )
+}
+GitHubRepoList.propTypes = PropTypes.func.isRequired
+
+const createProject = () => {}
+
+const NewProject = ({ handleGithubSignIn }) => {
+  const [ title, setTitle ] = useState('')
+  const [ description, setDescription ] = useState('')
+  const theme = useTheme()
+  const { appState: { github = { status: STATUS.INITIAL } } } = useAragonApi()
+
+  return (
+    <PanelContent>
+      <InfoBox>
+        Create a new project that belongs and operates entirely within this application, or synchronize one from GitHub.
+      </InfoBox>
+
+      {
+        // eslint-disable-next-line no-constant-condition
+        true ? (
+          <GitHubRepoList handleGithubSignIn={handleGithubSignIn} />
+        ) : (
+        <>
+      <FormField
```

Comment:
indentation seems way off

Reply:
Yes. Fixed it in case it's going to be needed for more than one day.